build.gradle.kts

```diff
@@ -25,43 +25,48 @@ java {
 }
 
 repositories {
+    if(project.properties["useMavenLocal"] == "true") {
+        logger.warn("Using local Maven repository as source")
+        mavenLocal()
+    }
     mavenCentral()
+    maven("https://oss.sonatype.org/content/repositories/graphicsscenery-1223")
```

Comment:
I don't think @ctrueden will like this. @ctrueden IIRC you could do something on the scijava maven so we wouldn't need to do this.

Reply:
That looks like something that @skalarproduktraum did just for testing? Because the 1223 is a particular staging repo, isn't it? Not something long term?

Reply:
Based on https://search.maven.org/search?q=g:graphics.scenery%20AND%20a:scenery%20AND%20v:0.9.0, it seems that scenery 0.9.0 is not actually released to Maven Central yet... but once it is, we won't need this extra line anyway. Let's definitely purge it from this PR.

Reply:
This was not intended to stay, but just to be able to use the 0.9.0 binaries from the staging repo 👍